### DISCUSSION TITLE
Training on resolutions higher than 512

### DISCUSSION THREAD

**Discussion by gilroff** [TheLastBen/fast-stable-diffusion#1750] [Q&A] [answered]
Can I make 512 res images even if i trained on higher resolution?

**Comment by TheLastBen** [comment, marked as answer]
if you train the model on 640px for example, if you generate at 512, it will give slightly lower quality, so you should always stay equal or higher than the trained res

**Reply by gilroff** [reply]
ok got it thanks.
